packages/dx-react-scheduler/src/types/views/basic-view.types.ts

```diff
@@ -0,0 +1,50 @@
+import { ViewCell } from '../index';
+
+/* tslint:disable no-namespace max-line-length no-empty-interface */
+/** @internal */
+export interface BasicViewProps {
```

Comment:
`any` types should be replaced with some interfaces 
Seems that `VerticalViewProps` and `BasicViewProps` are almost the same, we can derive `VerticalViewProps` from `BasicViewProps` or both from some common interface